resources/definitions/fokoos_base.def.json

```diff
@@ -0,0 +1,215 @@
+{
+    "name": "Fokoos Base Printer",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": false,
+        "author": "fokoos.com",
+        "manufacturer": "FOKOOS",
+        "file_formats": "text/x-gcode",
+        "first_start_actions": ["MachineSettingsAction"],
+        "machine_extruder_trains": { "0": "fokoos_extruder_1"},
+        "has_materials": true,
+        "preferred_material": "generic_pla",
+
+        "has_variants": true,
+        "variants_name": "Nozzle Size",
+        "preferred_variant_name": "0.40mm_fokoos_nozzle", 
+        "has_machine_quality": true,
+        "preferred_quality_type": "fokoos_layer_020",
+        "exclude_materials": [
+            "dsm_arnitel2045_175",
+            "dsm_novamid1070_175",
+            "emotiontech_asax",
+            "emotiontech_bvoh",
+            "emotiontech_copa",            
+            "emotiontech_hips",
+            "emotiontech_nylon_1030",            
+            "emotiontech_nylon_1030cf",    
+            "emotiontech_nylon_1070",   
+            "emotiontech_pc",    	
+            "emotiontech_pekk",    													 						        
+            "emotiontech_pva-m",
+            "emotiontech_pva-oks",
+            "emotiontech_pva-s",
+            "eSUN_PLA_PRO_Black",
+            "eSUN_PLA_PRO_Grey",
+            "eSUN_PLA_PRO_Purple",
+            "eSUN_PLA_PRO_White",
+            "Extrudr_GreenTECPro_Anthracite_175",    
+            "Extrudr_GreenTECPro_Black_175",  
+            "Extrudr_GreenTECPro_Blue_175",     
+            "Extrudr_GreenTECPro_Nature_175",  
+            "Extrudr_GreenTECPro_Red_175",       
+            "Extrudr_GreenTECPro_Silver_175",   
+            "Extrudr_GreenTECPro_White_175",           
+            "goofoo_asa",   
+            "goofoo_hips",
+            "goofoo_pa",
+            "goofoo_pa_cf",
+            "goofoo_pc",
+            "goofoo_peek",
+            "goofoo_pva",
+            "goofoo_tpe_83a",
+            "goofoo_tpu_87a",
+            "goofoo_tpu_95a",
+            "goofoo_wood_pla",                                                                                                              
+            "innofill_innoflex60_175",
+            "leapfrog_epla_natural",
+            "leapfrog_pva_natural",
+            "redd_asa",
+            "redd_hips",
+            "redd_nylon",
+            "redd_tpe",            
+            "structur3d_dap100silicone",
+            "tizyx_flex",
+            "tizyx_petg",
+            "tizyx_pla",
+            "tizyx_pla_bois",
+            "tizyx_pva",
+            "ultimaker_bam",
```

Comment:
There is no need to exclude all the ultimaker materials as this printer is a 1.75 mm printer. Ultimaker materials are 2.85mm and are filtered out by default.

Reply:
I have checked ultimaker materials, you are right.